### ISSUE TITLE
optical flow

### ISSUE THREAD

**Issue by MeNinaa** [cxjyxxme/deep-online-video-stabilization#3]
Hi,

you mention in your readme that optical flow is set to 0. May I ask you how to use flow information during training please?

Thank you

**Comment by cxjyxxme** [comment]
Hi, set flow to 0 is done during generating tfrecord datas, you can run the training scripts to train the net directly.

**Comment by MeNinaa** [comment]
Hi, 

thanks for replying. I can train the network now, however I would like to use flow information in the training process. For now, the losses I am getting are very high (~100), so I think the lack of flow information might be a cause.

Can you please explain how to generate the tfrecord data? 

Cheers

**Comment by cxjyxxme** [comment]
Hi,
I have trained with this tfrecord actually, and the final loss is about 5. This is the training record I trained, hope it's helpful.
![)OL9 EPY11K6LV5NEIMCLE0](https://user-images.githubusercontent.com/8403051/55332480-112f0f80-54c8-11e9-85f4-9aea8efc22d1.png)

**Comment by MeNinaa** [comment]
Hi,

I restarted training and now my losses seem to be similar to yours, thanks. This figure was really helpful.

But this procedure is for using the training data you provided, right? If I want to train using my own data, will I need to generate my own tf record data?

Cheers

**Comment by thedjofsis** [comment]
> Hi,
> 
> I restarted training and now my losses seem to be similar to yours, thanks. This figure was really helpful.
> 
> But this procedure is for using the training data you provided, right? If I want to train using my own data, will I need to generate my own tf record data?
> 
> Cheers

Hello @mariaito ,
I'm trying to work with this code and so far figured out the data directory structures by trial and error. Now I'm stuck with this same question about tfrecords for my own data. Have you been able to figure it out?

@cxjyxxme  If you can guide us for this step it will be very helpful,

Thank you
Regards,
Dharm

**Comment by A-Kerim** [comment]
Hi @mariaito @cxjyxxme.

I would like to train your model using my own data. Could you please direct us on how to generate the tfrecords?

Thanks,
Kerim
